### PR TITLE
Respective changes as result of released windows-instance module

### PR DESCRIPTION
**Pull request by sergiimatusEPAM**
1) Added correct Docker image with Ansible changes for Windows Agent 
2) Removed Terraformfile 
3) Updated README.md to avoid link to custom Mesosphere's `./terraform` package

### CODE REVIEW

**Comment by sergiimatusEPAM** [comment]
@sebbrandt87 , @fatz , please review/comment/approve, once you have free time

File: .gitignore [hunk 1]
```diff
@@ -10,3 +10,6 @@
 
 # ansible inventory
 inventory
+
+# IDE
+**/.idea/*
```
Comment (sebbrandt87) [inline .gitignore:15]: missing line break

File: README.md [hunk 2]
```diff
@@ -6,6 +6,7 @@ For production usage make sure you understand how terraform works and stores its
 ## Examples
 
 - AWS
+  - [Windows-agent](aws/windows-agent) - An  example with  windows agent support
```
Comment (sebbrandt87) [inline README.md:9]: please move it to the end of the AWS list

File: aws/windows-agent/README.md [hunk 3]
```diff
@@ -1,20 +1,16 @@
...
-
-wait for the cluster to be spawned.
-It will also spin up a windows instance and uses the `ansible_bundled_container` docker image.
+The proper image can be found at [mesosphere/dcos-ansible-bundle](https://hub.docker.com/r/mesosphere/dcos-ansible-bundle/tags)
+For instance, `ansible_bundled_container = "mesosphere/dcos-ansible-bundle:feature-windows-support-039d79d"` 
 
 The `ansible_bundled_container` can be replaced with your own provided docker image, to test the ansible run
-which is fully integrated into the terraform apply process.
-It will be pulled onto the bootstrap node and being executed there.
+which is fully integrated into the terraform apply process.
```
Comment (sebbrandt87) [inline aws/windows-agent/README.md:15]: extra space?
also what changed here in the lines? 😕

File: aws/windows-agent/main.tf [hunk 4]
```diff
@@ -8,16 +8,12 @@ data "http" "whatismyip" {
 }
 
 locals {
-  cluster_name = "generic-dcos-ee-demo"
+  cluster_name = "generic-dcos-demo"
 }
 
 module "dcos" {
   source  = "dcos-terraform/dcos/aws"
-  version = "~> 0.2.0"
```
Comment (sebbrandt87) [inline aws/windows-agent/main.tf:16]: the pessimistic version constraint is correct

File: aws/windows-agent/main.tf [hunk 5]
```diff
@@ -8,16 +8,12 @@ data "http" "whatismyip" {
...
-
```
Comment (sebbrandt87) [inline aws/windows-agent/main.tf:17]: please keep these lines as is

File: aws/windows-agent/main.tf [hunk 6]
```diff
@@ -35,19 +31,20 @@ module "dcos" {
...
   dcos_superuser_password_hash = "$6$rounds=656000$YSvuFmasQDXheddh$TpYlCxNHF6PbsGkjlK99Pwxg7D0mgWJ.y0hE2JKoa61wHx.1wtxTAHVRHfsJU9zzHWDoE08wpdtToHimNR9FJ/"
   dcos_superuser_username      = "demo-super"
 
-  additional_windows_private_agent_ips       = ["${concat(module.winagent.private_ips)}"]
-  additional_windows_private_agent_passwords = ["${concat(module.winagent.windows_passwords)}"]
+  additional_windows_private_agent_ips       = ["${concat(module.windows-instance.private_ips)}"]
+  additional_windows_private_agent_passwords = ["${concat(module.windows-instance.windows_passwords)}"]
 }
 
-module "winagent" {
```
Comment (sebbrandt87) [inline aws/windows-agent/main.tf:48]: the module name here is `winagent` and should stay like this entirely.
a module name does not need to match the source it is created from.

File: aws/windows-agent/main.tf [hunk 7]
```diff
@@ -35,19 +31,20 @@ module "dcos" {
...
+module "windows-instance" {
   source  = "dcos-terraform/windows-instance/aws"
-  version = "~> 0.2.1"
+  version = ">= 0.0.1"
```
Comment (sebbrandt87) [inline aws/windows-agent/main.tf:47]: a pessimistic version constraint like `~> 0.0.1` would match any "alpha" version and would exclude higher versions like `0.1.0` or `0.2.0` etc.

File: aws/windows-agent/README.md [hunk 8]
```diff
@@ -1,20 +1,16 @@
...
 
-## Terraform usage
+## Usage
+In order to get Windows instance span up, use latest `main.tf` and specify number of Windows Agents at `num` field.
+Currently `main.tf` contains `ansible_bundled_container` parameter which refers 
+to docker image with windows-related changes (tag `feature-windows-support-${commit}` indicates about it).
 
-You need our Terraformfile based terraform 0.11 fork: https://github.com/fatz/terraform/releases/tag/v0.11.14-mesosphere
-
-```
```
Comment (sebbrandt87) [inline aws/windows-agent/README.md:10]: I would say keep the spin up instructions from line incl 10 to line incl 16.

File: aws/windows-agent/README.md [hunk 9]
```diff
@@ -1,20 +1,16 @@
-# ALPHA STATE - DO NOT USE THIS EXAMPLE
+# Windows Agent support (Alpha state example)
 
 This is an early example how to use and integrate windows agents. If you're not
 an active developer of this project we do not suggest to use it.
 
-## Terraform usage
+## Usage
+In order to get Windows instance span up, use latest `main.tf` and specify number of Windows Agents at `num` field.
```
Comment (sebbrandt87) [inline aws/windows-agent/README.md:7]: In order to spin up a DC/OS cluster with extra Windows agents, use the latest `main.tf` and specify number of Windows Agents at `num` field.

**Review by sebbrandt87 — CHANGES_REQUESTED** [review]

**Comment by sebbrandt87** [comment]
updated feature branch

**Pull request closed without merging**